index.html

```diff
@@ -10,7 +10,12 @@
     <!-- Links and scripts -->
     <link href="/css/index.css" rel="stylesheet" />
```

Comment:
This line needs to be removed from all HTML files, since Vite is now taking care of it through index.js